signer-service/src/api/routes/v1/stellar.route.js

```diff
@@ -8,4 +8,9 @@ router.route('/create').post(validateCreationInput, controller.createStellarTran
 
 router.route('/payment').post(validateChangeOpInput, controller.changeOpTransaction);
 
+// TODO maybe add validator.
```

Comment:
These are the temporary endpoints. `GET` will return the public key, required to start the `sep-10` on the first place. `POST` will provide the signature.

I will leave this comment about validators, but I don't think is of too much value to add one now, considering these are only temporary.